### PR TITLE
Update shelter for JSFE-2023Q1

### PR DESCRIPTION
**Pull request by WiiJoy**

### CODE REVIEW

File: tasks/shelter/shelter.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+
+## Ключевые навыки:
+- валидная семантическая адаптивная вёрстка
+- легкоподдерживаемый читаемый код
+- экспорт стилей и графики из Figma
+- использование JavaScript для реализации указанного в задании функционала
+
+## Этапы работы над проектом:
+Задание состоит из трёх частей:
+- Часть 1. Фиксированная вёрстка
```
Comment (valerydluski) [inline tasks/shelter/shelter.md:16]: can we make the 'Часть 1' as a link?

**Review by valerydluski — COMMENTED** [review]
please update stage1 readme

File: tasks/shelter/shelter.md (hunk 1 above)
Comment (WiiJoy) [reply tasks/shelter/shelter.md]: Ok, and rename this to 'shelter. week-N' as in schedule

**Comment by WiiJoy** [comment]
> 

updated

**Review by valerydluski — APPROVED** [review]

**Pull request merged**